<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shoufacun_info</class>
 <widget class="QDialog" name="shoufacun_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>收发存统计</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>881</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>881</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>筛选条件</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>861</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>存货编码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>存货名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>规格型号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="spec_lineEdit">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>71</y>
      <width>351</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>操作日期范围(含):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="st_dateEdit">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>——</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="ed_dateEdit">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="query_btn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>筛选</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_query_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>清除筛选</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>110</y>
      <width>851</width>
      <height>111</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;1.初始列出&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;所有存货&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#000000;&quot;&gt;的收发存数据&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;2.&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;存货编码&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;、&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;存货名称&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;、&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;规格型号&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;均&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;模糊搜索&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;3.某项多个输入用&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;;(英文分号)&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;隔开，例如：ID1&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;;&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;ID2&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400; color:#0000ff;&quot;&gt;;&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;ID3&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;4.期初数量对应开始日期当天所有操作之前；&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;5.结存数量对应截止日期当天所有操作结束。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="export_excel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>590</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>导出表中数据至Excel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>590</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
